src/test/java/com/google/graphgeckos/dashboard/DashboardApplicationTests.java

```diff
@@ -1,6 +1,6 @@
 package com.google.graphgeckos.dashboard;
 
-import org.junit.jupiter.api.Test;
+// import org.junit.jupiter.api.Test;
```

Comment:
just delete this line if it is not required for the code to compile for now.

Reply:
Done!